spec/fixtures/cytoskeletal_metadata.json

```diff
@@ -150,8 +150,8 @@
     "related_objects": [
         {
             "related_identifier": "https://www.biorxiv.org/content/10.1101/545517v1",
-            "related_identifier_type": "arXiv",
-            "relation_type": "isCitedBy"
+            "related_identifier_type": "ARXIV",
+            "relation_type": "IS_CITED_BY"
```

Comment:
This surprises me, a little: I would think the all-caps versions are just internal keys generated for the ruby library. I may be thinking about it wrong.